index.js

```diff
@@ -14,8 +14,10 @@ const exec = require('@actions/exec')
 
 try {
   const os = core.getInput('os')
+  const version = core.getInput('version')
   console.log(" OS - " + os)
-   runCommand(os)
+  console.log(" Version - " + version)
```

Comment:
This can move to `if(version)` block, so only if we have version specified we log something like 'aio cli version .... '